.idea/codeStyles/Project.xml

```diff
@@ -227,5 +227,8 @@
         </rules>
       </arrangement>
     </codeStyleSettings>
+    <codeStyleSettings language="kotlin">
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
+    </codeStyleSettings>
   </code_scheme>
-</component>
\ No newline at end of file
+</component>
```

Comment:
As mentioned on slack and noted earlier on #15 - could please roll that back?
It slipped my re-review.